exercises/launch-matrix-transpose-local-array_solution.cpp

```diff
@@ -374,20 +384,23 @@ int main(int RAJA_UNUSED_ARG(argc), char **RAJA_UNUSED_ARG(argv[]))
             });
           });
 
-          RAJA::loop_icount<hip_threads_x>(ctx, col_tile, [&] (int col, int tx) {
-           RAJA::loop_icount<hip_threads_y>(ctx, row_tile, [&] (int row, int ty) {
+          ctx.teamSync();
+
+          RAJA::loop_icount<hip_threads_y>(ctx, col_tile, [&] (int row, int ty) {
+            RAJA::loop_icount<hip_threads_x>(ctx, row_tile, [&] (int col, int tx) {
 
-             d_Atview(col, row) = Tile_Array[ty][tx];
+              d_Atview(row, col) = Tile_Array[tx][ty];
```

Comment:
@MrBurmark , I switched it around so its clear that the x and y threads have been transposed in shared memory. I'm not too sure how to express that in Kernel.

Reply:
using row_t and col_t was the source of the compilation error. I switched it back so code compiles. Please feel free to undo my change and fix the compile differently.